juriscraper/OpinionSiteAspx.py

```diff
@@ -0,0 +1,74 @@
+from lxml import html
+from juriscraper.OpinionSite import OpinionSite
+
+
+class OpinionSiteAspx(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(OpinionSiteAspx, self).__init__(*args, **kwargs)
+        self.spoof_user_agent = False
+
+    def _get_soup(self, url):
```

Comment:
this looks awesome!  my only extremely minor gripe is that perhaps this method should not be prefixed with `_get`, since it doesn't actually return anything. Should it be `_set`, `_update`, `_refresh` or something else instead?

Reply:
Done